logos/benchmarks/benchmark_logos.py

```
@@ -0,0 +1,1240 @@
+            return None
+        combined.sort()
+        energy_j = 0.0
+        for i in range(1, len(combined)):
+            t0, p0 = combined[i - 1]
+            t1, p1 = combined[i]
+            energy_j += (p0 + p1) / 2.0 / 1000.0 * (t1 - t0)
+        return energy_j
+
+    def power_samples(self) -> list[tuple[float, float]]:
+        combined: list[tuple[float, float]] = []
+        for samples, lock in zip(self._host_samples, self._locks):
```

Ruff (0.15.15)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)